src/abi/interaction.spec.ts

```diff
@@ -251,20 +252,17 @@ describe("test smart contract interactor", function () {
         await provider.sendTransaction(transaction);
         assert.equal(transaction.getNonce().valueOf(), 0n);
         assert.equal(transaction.getData().toString(), "getUltimateAnswer");
-        assert.equal(
-            transaction.getHash().toString(),
-            "3579ad09099feb9755c860ddd225251170806d833342e912fccdfe2ed5c3a364",
-        );
+        const computer = new TransactionComputer();
```

Comment:
Maybe can hold result of `await provider.sendTransaction(transaction);` and don't use this computer? (serves as a more real-world example, I think).

Reply:
Below, as well.